aw_watcher_window/config.py

```diff
@@ -1,18 +1,31 @@
 from configparser import ConfigParser
+import argparse
 
 from aw_core.config import load_config as _load_config
 
-
 def load_config():
     default_client_config = ConfigParser()
-    default_client_config["aw-watcher-window"] = {
-        "exclude_title": False,
-        "poll_time": "1.0"
-    }
-    default_client_config["aw-watcher-window-testing"] = {
+    default_client_config["aw-watcher-window"] = default_client_config["aw-watcher-window-testing"] = {
         "exclude_title": False,
-        "poll_time": "1.0"
+        "poll_time": "1.0",
+        "macos_strategy": "jxa"
```

Comment:
```suggestion
        "strategy": "jxa"
```

Reply:
Did this change work without you having to manually edit the config file? (we've had some issues where new defaults won't be applied for old installs...)

Reply:
I now realize why you chose to prepend `macos_` to the strategy...

Reply:
I'm confused here: do you want the var name to change before merging?

Reply:
No sorry, I already applied the suggested change :)